src/components/family-search/family-search-results-table/FamilySearchResultsTable.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import {
+  Box,
+  Paper,
+  Table,
+  TableHead,
+  TableBody,
+  TableRow,
+  TableCell,
+  TableFooter,
+  TableContainer,
+} from "@material-ui/core";
+import { makeStyles } from "@material-ui/styles";
+import { FamilySearchResponse } from "../../../api/FamilyAPI";
+import { DefaultFields } from "../../../constants/DefaultFields";
+
+const useStyles = makeStyles(() => ({
+  noResultsTableCell: {
+    textAlign: "center",
+  },
+  tableContainer: {
+    maxHeight: 250,
+  },
+}));
+
+type Props = {
+  families: FamilySearchResponse[];
+};
+
+const FamilySearchResultsTable = ({ families }: Props) => {
+  const classes = useStyles();
+  return (
+    <Box marginY={2}>
+      <TableContainer
+        component={Paper}
+        elevation={0}
+        className={classes.tableContainer}
+      >
+        <Table stickyHeader aria-label="family search results table">
+          <caption hidden>Family search results</caption>
+          <TableHead>
+            <TableRow>
+              <TableCell>{DefaultFields.FIRST_NAME.name}</TableCell>
+              <TableCell>{DefaultFields.LAST_NAME.name}</TableCell>
+              <TableCell>{DefaultFields.PHONE_NUMBER.name}</TableCell>
+              <TableCell>{DefaultFields.EMAIL.name}</TableCell>
+              <TableCell>{DefaultFields.NUM_CHILDREN.name}</TableCell>
```

Comment:
Maybe we can replace this with something like
```suggestion
              {DefaultFamilySearchFields.map((searchField) => (
                <TableCell>{searchField.name}</TableCell>
              ))}
```